waves/src/wasmProxy.ts

```diff
@@ -40,3 +40,8 @@ export async function getBalances(): Promise<BalanceEntry[]> {
     const { get_balances } = await import("./wallet");
     return get_balances(WALLET_NAME);
 }
+
+export async function withDrawAll(address: string): Promise<String> {
```

Comment:
I would prefer to keep the names consistent everywhere but at the very least, it should be `withdraw` :)